lib/apis/v3/felixconfig.go

```diff
@@ -113,22 +110,22 @@ type FelixConfigurationSpec struct {
 	// “INPUT” chain; Calico will insert its rules at the top of that chain, then “RETURN” packets to the “INPUT” chain
 	// once it has completed processing workload endpoint egress policy. Use ACCEPT to unconditionally accept packets
 	// from workloads after processing workload endpoint egress policy. [Default: DROP]
-	DefaultEndpointToHostAction string `json:"defaultEndpointToHostAction,omitempty"`
-	IptablesFilterAllowAction   string `json:"iptablesFilterAllowAction,omitempty"`
-	IptablesMangleAllowAction   string `json:"iptablesMangleAllowAction,omitempty"`
+	DefaultEndpointToHostAction string `json:"defaultEndpointToHostAction,omitempty" validate:"omitempty,dropacceptreturn"`
+	IptablesFilterAllowAction   string `json:"iptablesFilterAllowAction,omitempty" validate:"omitempty,acceptreturn"`
+	IptablesMangleAllowAction   string `json:"iptablesMangleAllowAction,omitempty" validate:"omitempty,acceptreturn"`
 	// LogPrefix is the log prefix that Felix uses when rendering LOG rules. [Default: calico-packet]
 	LogPrefix string `json:"logPrefix,omitempty"`
 
 	// LogFilePath is the full path to the Felix log. Set to none to disable file logging. [Default: /var/log/calico/felix.log]
 	LogFilePath string `json:"logFilePath,omitempty"`
 
 	// LogSeverityFile is the log severity above which logs are sent to the log file. [Default: INFO]
-	LogSeverityFile string `json:"logSeverityFile,omitempty"`
+	LogSeverityFile string `json:"logSeverityFile,omitempty" validate:"omitempty,felixLogLevel"`
```

Comment:
nit: Our validation tags are a mix of all lower and camelcase.  Would be nice to choose one or t'other.

Reply:
will fix